docker/constants.dev.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+GATEWAY_URL: str = "http://unstable--dummy-api.libero.pub"
+ELASTICSEARCH_HOSTS: List[str] = [
+    "http://elasticsearch:9200"
+]
+CONTENT_SERVICES_TO_INDEX: List[str] = [
+    "scholarly-articles",
+    "blog-articles"
+]
```

Comment:
seeing the configuration file format has changes from JSON to Python code; sounds like a thing to document in the README? (Notwithstanding the usefulness of this file as an example that is continuously tested)

Reply:
Agreed. I'll update the README once the approach is agreed.

Reply:
README updated.